homeassistant/components/sensibo/binary_sensor.py

```diff
@@ -87,6 +87,40 @@ class SensiboDeviceBinarySensorEntityDescription(
     ),
 )
 
+PURE_SENSOR_TYPES: tuple[SensiboDeviceBinarySensorEntityDescription, ...] = (
+    SensiboDeviceBinarySensorEntityDescription(
+        key="pure_boost_enabled",
+        device_class=BinarySensorDeviceClass.RUNNING,
+        name="Pure Boost Enabled",
+        icon="mdi:wind-power-outline",
+        value_fn=lambda data: data.pure_boost_enabled,
+    ),
+    SensiboDeviceBinarySensorEntityDescription(
+        key="pure_ac_integration",
+        device_class=BinarySensorDeviceClass.CONNECTIVITY,
+        name="Pure Boost linked with AC",
+        icon="mdi:connection",
+        value_fn=lambda data: data.pure_ac_integration,
+        entity_registry_enabled_default=False,
+    ),
+    SensiboDeviceBinarySensorEntityDescription(
+        key="pure_geo_integration",
+        device_class=BinarySensorDeviceClass.CONNECTIVITY,
+        name="Pure Boost linked with Presence",
+        icon="mdi:connection",
+        value_fn=lambda data: data.pure_geo_integration,
+        entity_registry_enabled_default=False,
+    ),
+    SensiboDeviceBinarySensorEntityDescription(
+        key="pure_measure_integration",
+        device_class=BinarySensorDeviceClass.CONNECTIVITY,
+        name="Pure Boost linked with Indoor Air Quality",
+        icon="mdi:connection",
+        value_fn=lambda data: data.pure_measure_integration,
+        entity_registry_enabled_default=False,
+    ),
```

Comment:
Should these be `entity_category=EntityCategory.DIAGNOSTIC,`?

Reply:
I think probably it's a better fit and I could leave them enabled actually as they won't be added to a dashboard automatically anyway.

Reply:
I am going to add in the Pure Boost Service as well but I can do that in follow-up PR if you are happy with this one and we merge it.